src/cascadia/TerminalApp/Profile.cpp

```diff
@@ -187,69 +187,66 @@ Json::Value Profile::ToJson() const
     Json::Value root;
 
     ///// Profile-specific settings /////
-    root[GUID_KEY.data()] = winrt::to_string(Utils::GuidToString(_guid));
-    root[NAME_KEY.data()] = winrt::to_string(_name);
+    root[JsonKey(GUID_KEY)] = winrt::to_string(Utils::GuidToString(_guid));

```

Comment:
do we need a `GuidToString<T>`?
also, `winrt::to_string` (or to_hstring? ugh i don't know) actually has a guid overload! `to_string(_guid)`?

Reply:
to_hstring has a winrt::guid overload, so this ends up becoming
```
jsonObject[JsonKey(DEFAULTPROFILE_KEY)] = winrt::to_string(winrt::to_hstring(winrt::guid{ _defaultProfile }));
```

which seems fundamentally worse